providers/fab/src/airflow/providers/fab/www/views.py

```diff
@@ -77,22 +75,11 @@ def index(self):
 
 def show_traceback(error):
     """Show Traceback for a given error."""
-    is_logged_in = get_auth_manager().is_logged_in()
     return (
         render_template(
             "airflow/traceback.html",
-            python_version=sys.version.split(" ")[0] if is_logged_in else "redacted",
-            airflow_version=version if is_logged_in else "redacted",
-            hostname=(
-                get_hostname()
-                if conf.getboolean("webserver", "EXPOSE_HOSTNAME") and is_logged_in
-                else "redacted"
-            ),
-            info=(
```

Comment:
We should also axe this then:
https://github.com/apache/airflow/blob/545801ce2609610002c0b2b290109c984f43a5bd/providers/fab/src/airflow/providers/fab/www/templates/airflow/traceback.html#L51-L53

Reply:
Updated